ProjFS.Mac/PrjFSKextLogDaemon/PrjFSKextLogDaemon.cpp

```diff
@@ -296,12 +312,11 @@ static void LogKextMessage(os_log_type_t messageLogType, uint32_t messageFlags,
 {
     if (messageFlags & LogMessageFlag_LogMessagesDropped)
     {
-        // One or more earlier messages have been dropped
-        os_log_error(s_kextLogger, "One or more kext log messages have been dropped");
-        
-        JsonWriter messageWriter;
-        messageWriter.Add(MessageKey, "One or more kext log messages have been dropped");
-        WriteJsonToMessageListener(MessageType::Error, messageWriter);
+        if (1 == ++s_droppedMessageCount)
```

Comment:
I was debating if we needed a log on the first message and a total count every 30 minutes, but i think it's not too much noise and a good call.
If we only did every 30 minutes we may message a message if the Kext is unloaded in that block of time.

Reply:
Yeah I went back and forth on the same thing, but I decided I didn't want us to ever lose that a message was dropped.

As you pointed out if we only logged in the timer and things are shut down between it fires we would miss that a message was dropped.